src/zrc/api/viewsets.py

```diff
@@ -191,8 +198,12 @@ def _zoek(self, request, *args, **kwargs):
         return self.get_search_output(queryset)
     _zoek.is_search_action = True
 
+    def get_kenmerken(self, data):
+        return [{k: data.get(k, '')} for k in settings.NOTIFICATIES_KENMERKEN_NAMES]
+
 
-class StatusViewSet(CheckQueryParamsMixin,
+class StatusViewSet(NotificationCreateMixin,
```

Comment:
I think using the ViewSetMixin is the better choice over CreateMixin, to prevent future errors.

Reply:
In this case I will add update() and destroy() actions to viewsets which don't support this actions.